src/realizations/catchment/Bmi_Multi_Formulation.cpp

```diff
@@ -292,6 +292,11 @@ string Bmi_Multi_Formulation::get_output_line_for_timestep(int timestep, std::st
         throw std::invalid_argument("Only current time step valid when getting multi-module BMI formulation output");
     }
 
+    // check if a requested output variable name is valid, if not, stop the execution
+    if (timestep == 0) {
+        check_output_var_names();
```

Comment:
I think this will work if this function is called at the end of `create_multi_formulation` instead of in the time loop... can you try it there instead?